### PR TITLE
Add a warning to the console to discourage attacks and encourage contributing

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-web/issues/2803
Replaces https://github.com/matrix-org/matrix-react-sdk/pull/7608
Type: enhancement

<hr>

---
Here's what your changelog entry will look like:

## ✨ Features
 * Add a warning to the console to discourage attacks and encourage contributing ([\#7673](https://github.com/matrix-org/matrix-react-sdk/pull/7673)). Fixes vector-im/element-web#2803. Contributed by @SimonBrandner.

### CODE REVIEW

File: src/components/structures/MatrixChat.tsx [hunk 1]
```diff
@@ -501,6 +506,34 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         });
     }
 
+    private onWindowResized = (): void => {
+        // XXX: This is a very unreliable way to detect whether or not the the devtools are open
+        this.warnInConsole();
+    };
+
+    private warnInConsole(): void {
```
Comment (t3chguy) [inline src/components/structures/MatrixChat.tsx:514]: can this get a leading&trailing throttle on it so it doesn't get spammed during slow(long) resize operations

File: src/components/structures/MatrixChat.tsx [hunk 2]
```diff
@@ -120,6 +120,8 @@ import AccessibleButton from "../views/elements/AccessibleButton";
 import { ActionPayload } from "../../dispatcher/payloads";
 import { SummarizedNotificationState } from "../../stores/notifications/SummarizedNotificationState";
 
+import { throttle } from "lodash";
```
Comment (t3chguy) [inline src/components/structures/MatrixChat.tsx:123]: please put absolute imports above relative ones :'(

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/components/structures/MatrixChat.tsx (hunk 2 above)
Comment (SimonBrandner) [reply src/components/structures/MatrixChat.tsx:123]: Argh, VSCode failed me :D

**Pull request merged**